<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Traffic Accident in USA  Predictor</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>631</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_loading">
     <attribute name="title">
      <string>Loading</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_loading">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Loading Data</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_output">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>591</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>137</x>
        <y>22</y>
        <width>411</width>
        <height>111</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_eda">
     <attribute name="title">
      <string>EDA</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_EDA">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>112</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>EDA Plot</string>
      </property>
     </widget>
     <widget class="MplWidget" name="MplWidget" native="true">
      <property name="geometry">
       <rect>
        <x>69</x>
        <y>70</y>
        <width>611</width>
        <height>361</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>300</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_EDA2">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>10</y>
        <width>112</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>EDA Plot 2</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_preprocessing">
     <attribute name="title">
      <string>Preprocessing</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>31</y>
        <width>611</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_features">
     <attribute name="title">
      <string>Feature Selection</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_modeling">
     <attribute name="title">
      <string>Modeling</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit_2"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionExit_2">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header location="global">mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
